src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i < figures.length; i++) {
+            if (i < figures.length / 2) {
+                figureSupplier.getRandomFigure().printInformation();
```

Comment:
Instead of calling `printInformation()` directly in the loop, you should assign the created figures to the `figures` array elements. Then, after the loop, you can iterate over the array and call `printInformation()` on each figure.